Pipfile

```diff
@@ -0,0 +1,18 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+docker = "==7.1.0"
+jsonrpcbase = "==0.2.0"
```

Comment:
Looks like gitpython is missing?

Reply:
Do we need really gitpython?

Reply:
I don't know. Since it was in the requirements, my assumption is that it's needed until it can be shown that's not the case.

Reply:
I did a global search and did not find import git anywhere.

Reply:
could be a `from git import ...` style import

Reply:
Nothing showed up either.

Reply:
Have you read the code that interacts with git repos? How does it work?

Reply:
They use subprocess https://github.com/kbase/catalog/blob/283b022e26336d497a6322112093bf7fda8b6ed7/lib/biokbase/catalog/registrar.py#L82C13-L82C23

Reply:
:+1: